chaoslib/litmus/disk_fill/disk_fill_by_litmus.yml

```diff
@@ -0,0 +1,135 @@
+- name: Get a Random Pod name
+  shell: >
+    kubectl get pods -l {{ a_label }} -n {{ a_ns }} --no-headers -o custom-columns=:metadata.name | shuf -n 1 
+  args:
+    executable: /bin/bash
+  register: pod_name
+
+- name: Get ephemeral-storage Limit Value
+  shell: >
+    kubectl get pods {{ pod_name.stdout }} -n {{ a_ns }} -o jsonpath='{.spec.containers[?(@.name=="{{ c_container }}")].resources.limits.ephemeral-storage}' | grep -Eo '+|[0-9]+'
+  args:
+    executable: /bin/bash
+  register: limit_value
+
+- name: Get ephemeral-storage Limit storage unit
+  shell: >
+    kubectl get pods {{ pod_name.stdout }} -n {{ a_ns }} -o jsonpath='{.spec.containers[?(@.name=="{{ c_container }}")].resources.limits.ephemeral-storage}' | grep -Eo '[[:alpha:]]+'
+  args:
+    executable: /bin/bash
+  register: limit_value_storage
+
+- name: Get ephemeral-storage Request Value
+  shell: >
+    kubectl get pods {{ pod_name.stdout }} -n {{ a_ns }} -o jsonpath='{.spec.containers[?(@.name=="{{ c_container }}")].resources.requests.ephemeral-storage}' | grep -Eo '+|[0-9]+'
+  args:
+    executable: /bin/bash
+  register: req_value
+
+- name: Get ephemeral-storage Request storage unit
+  shell: >
+    kubectl get pods {{ pod_name.stdout }} -n {{ a_ns }} -o jsonpath='{.spec.containers[?(@.name=="{{ c_container }}")].resources.requests.ephemeral-storage}' | grep -Eo '[[:alpha:]]+'
+  args:
+    executable: /bin/bash
+  register: req_value_storage
+
+
+- name: Fetch the value of Request ephemeral-storage Unit in KB
+  set_fact:
+    req_storage_KB: "{{ lookup('vars', req_value_storage.stdout) }}"
```

Comment:
Is there any benefits of using  lookup command ```lookup('vars', req_value_storage.stdout)```  here ,  Instead of  ``` req_storage_KB:  "{{req_value_storage.stdout) }}" ``` this  direct approach??. If we use same command for a given tasks in all experiments then it will looks good .